src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -30,24 +31,28 @@ class DeprecationErrorHandler
 
     private $mode;
     private $configuration;
-    private $deprecations = [
-        'unsilencedCount' => 0,
-        'remaining selfCount' => 0,
-        'legacyCount' => 0,
-        'otherCount' => 0,
-        'remaining directCount' => 0,
-        'remaining indirectCount' => 0,
-        'unsilenced' => [],
-        'remaining self' => [],
-        'legacy' => [],
-        'other' => [],
-        'remaining direct' => [],
-        'remaining indirect' => [],
-    ];
+    private $deprecationGroups = [];
```

Comment:
Too bad symfony team doesn't like documenting private props, phpstorm doesn't understand `$this->deprecationGroups[$group]->foo()` calls now

Reply:
I'm going to document it, good call 👍